mariner/file_formats/tests/test_cbddlp.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+import io
+import pathlib
+from unittest import TestCase
+
+import png
+from pyexpect import expect
+
+from mariner.file_formats.ctb import CTBFile
+
+
+class CTBFileTest(TestCase):
+    def test_loading_ctb_file(self) -> None:
```

Comment:
this should be called `test_loading_cdbdlp_file`

Reply:
code updated